api/envoy/service/auth/v3/external_auth.proto

```diff
@@ -50,9 +50,8 @@ message DeniedHttpResponse {
   // code to the downstream client other than 403 (Forbidden).
   type.v3.HttpStatus status = 1 [(validate.rules).message = {required: true}];
 
-  // This field allows the authorization service to send HTTP response headers
-  // to the downstream client. Note that the :ref:`append field in HeaderValueOption <envoy_v3_api_field_config.core.v3.HeaderValueOption.append>` defaults to
```

Comment:
@esmet sorry, that I don't remember all of the contexts why we put this note (and all similar notes below). I think since you implemented this for HTTP-based ext_authz, hence ignoring the `append` field value?

Reply:
Wait, I think I have been mistaken on this. Will update.